cvat-ui/src/components/signing-common/styles.scss

```diff
@@ -34,14 +34,18 @@ $social-google-background: #4286f5;
     }
 }
 
+.cvat-signing-header-logo-wrapper {
+    height: 100%;
+}
+
 .cvat-signing-header {
     background: transparent;
     position: fixed;
     padding: $grid-unit-size * 2 0 0 0;
     width: 100%;
 
     .cvat-logo-icon {
-        fill: white;
+        filter: brightness(0) invert(1);
```

Comment:
As I understand it makes any logo on sign-in page white, however what if a color logo provided? 
This approach will break such logo.